README.md

```diff
@@ -1,7 +1,9 @@
 # Pola iOS
 
+[![Build Status](https://travis-ci.org/KlubJagiellonski/pola-ios.svg?branch=master)](https://travis-ci.org/KlubJagiellonski/pola-ios)
+
 ## Chcesz pomóc?
 
 * Wpierw przeczytaj [CONTRIBUTING.md](https://github.com/KlubJagiellonski/pola-ios/blob/master/CONTRIBUTING.md)
 * [Sprawdź issues'y](https://github.com/KlubJagiellonski/pola-ios/issues)
-* Jeżeli czegoś nie wiesz lub też masz z czymś problem pisz na [kontakt@pola-app.pl](kontakt@pola-app.pl)
+* Jeżeli czegoś nie wiesz lub też masz z czymś problem pisz na [aplikacja.pola@gmail.com](aplikacja.pola@gmail.com)
```

Comment:
Czy ta zmiana została uzgodniona z @jakublipinski ?

Reply:
Robiłem to w innym PR ( https://github.com/KlubJagiellonski/pola-ios/pull/37 ) i approve od Kuby był. Tylko nie zauważyłem że email też jest w readme (robiłem grepa w Xcode po plikach dodanych do projektu)

Reply:
Approved